src/index.html

```diff
@@ -7,8 +7,74 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a class="logo" href="#Home">
+            <img src="../src/images/logo.png" alt="logo.png">
+          </a>
+        </div>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#apple">
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#samsung">
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#smartphones and phones">
+                Смартфоны и телефоны
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link"
+                href="#laptops and computers"
+                data-qa="hover"
+              >
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), the 3rd item